src/types/ReduxActions.ts

```diff
@@ -31,16 +31,13 @@ export type Action =
   | { type: 'CREATE_ACCOUNT_FAIL'; data: string /* error */ }
   | { type: 'CREATE_ACCOUNT_SUCCESS'; data: EdgeAccount }
   | {
+      // TODO: remove
       type: 'CREATE_UPDATE_PIN'
       data: {
         pin: string
         error: string | null
       }
     }
```

Comment:
Delete this action, along with the `CreateAccountActions.ts` file, which is now unused.